src/pages/guides/index.md

```diff
@@ -251,7 +251,8 @@ This feature is scheduled to be rolled out on 10th February, 2022
 For a more robust and reliable verification, Adobe I/O Events adds below security validations for events delivered to your webhook. 
 
 - sending additional field of recipient_client_id as part of your event payload
-- signing the event payloads digitally and sending those digital signatures as request headers to your webhook
+- signing all the event payloads digitally using a fixed public/private key pair generated by Adobe I/O Events and sending those digital signatures as request headers to your webhook
```

Comment:
Let's be consistent with the terminology in the third point.
```suggestion
- signing all the event payloads digitally using a fixed public/private key pair generated by Adobe I/O Events, and sending those digital signatures as webhook request headers
```

Reply:
done